test/tools/libtesteth/TestHelper.cpp

```diff
@@ -242,23 +244,69 @@ void replaceLLLinState(json_spirit::mObject& _o)
 	}
 }
 
-std::vector<boost::filesystem::path> getJsonFiles(boost::filesystem::path const& _dirPath, std::string const& _particularFile)
+std::vector<boost::filesystem::path> getFiles(boost::filesystem::path const& _dirPath, std::set<string> const _extentionMask, std::string const& _particularFile)
```

Comment:
Inconsistent namespace usage. Alias `boost::filesystem` as `fs` and use `std::string`.